.changeset/fuzzy-terms-push.md

```diff
@@ -0,0 +1,36 @@
+---
+'@web/browser-logs': minor
+'@web/config-loader': minor
+'@web/dev-server': minor
+'@web/dev-server-core': minor
+'@web/dev-server-esbuild': minor
+'@web/dev-server-hmr': minor
+'@web/dev-server-import-maps': minor
+'@web/dev-server-legacy': major
+'@web/dev-server-rollup': minor
+'@web/dev-server-storybook': minor
+'@web/parse5-utils': major
+'@web/polyfills-loader': major
+'@web/rollup-plugin-copy': minor
+'@web/rollup-plugin-html': major
+'@web/rollup-plugin-import-meta-assets': major
+'@web/rollup-plugin-polyfills-loader': major
+'@web/rollup-plugin-workbox': minor
```

Comment:
Almost, this is not published in the `@web` org.
```suggestion
'rollup-plugin-workbox': minor
```

Reply:
This is the only one that isn't in the `@web` scope? 😄